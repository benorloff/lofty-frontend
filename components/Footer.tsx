import { Container, Grid, Typography } from "@mui/material"

export default function Footer () {
    return (
        <Container maxWidth="lg" sx={{ mt: '50px', mb: '50px' }}>
            <Grid container rowGap={2} justifyContent="space-between">
                    <Typography>
                        {`Copyright ${new Date().getFullYear()} Catstagram.`}
                    </Typography>
                    <Typography>
                        Contact Us
                    </Typography>
            </Grid>
        </Container>
    )
}