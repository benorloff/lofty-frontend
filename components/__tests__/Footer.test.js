import Footer from '../Footer'
import { render, screen } from '@testing-library/react'
import "@testing-library/jest-dom"

describe('Footer Component', () => {

    it('Contains Contact Us', async () => {

        const { container } = render(<Footer />)

        expect(container).toMatchSnapshot();
        expect(screen.getByText("Contact Us")).toBeInTheDocument()
    })
})