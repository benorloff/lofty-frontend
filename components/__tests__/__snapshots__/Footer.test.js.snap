// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Contains Contact Us 1`] = `
<div>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1jhv0pg-MuiContainer-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container css-s9rcqg-MuiGrid-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
      >
        Copyright 2023 Catstagram.
      </p>
      <p
        class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
      >
        Contact Us
      </p>
    </div>
  </div>
</div>
`;
